test/integration_test.tsx

```diff
@@ -416,52 +399,40 @@ function addInteractiveTests(render: typeof mount) {
   });
 }
 
+const createDefaultAdapter = () => new Adapter();
+function setAdapter(createNewAdapter: () => Adapter) {
+  beforeEach(() => {
+    configure({ adapter: createNewAdapter() });
+  });
+
+  afterEach(() => {
+    configure({ adapter: createDefaultAdapter() });
+  });
+}
+
 describe('integration tests', () => {
   before(() => {
-    configure({ adapter: new Adapter() });
+    configure({ adapter: createDefaultAdapter() });
   });
 
   describe('"mount" rendering', () => {
     addStaticTests(mount);
     addInteractiveTests(mount);
 
-    it('supports simulating events on deep Components and elements', () => {
```

Comment:
I moved this test into a describe block that changes the global adapter to more closely match what a user would do. I discovered that overriding the adapter in the `mount` or `shallow` options doesn't necessarily change it for other method (e.g. `wrapper.debug()`). So for these option tests, I'm calling `configure` again to change the adapter globally